translate/filters/checks.py

```diff
@@ -55,14 +55,16 @@
         %(                          # initial %
         (?P<boost_ord>\d+)%         # boost::format style variable order, like %1%
         |
+        \|?                         # boost::format allows to enclose params in pipes, %|spec|
```

Comment:
If no tests are added for this feature I prefer to leave it out for now. If tests are added I prefer tests and this particular change in the regex to be put in a separate PR.